app/controllers/users_controller.rb

```diff
@@ -0,0 +1,25 @@
+class UsersController < ApplicationController
+  def show
+    @user = User.find_by id: params[:id]
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new params[:user]
+    if @user.save
+      flash[:success] = t "welcome_to_the_sample_app!"
```

Comment:
```suggestion
      flash[:success] = t "users.create.success"
```
nên theo cấu trúc ni nhé, chỉ file I18n nhe